Bootstrap/Podfile.lock

```diff
@@ -35,10 +34,10 @@ CHECKOUT OPTIONS:
     :git: https://github.com/facebook/ios-snapshot-test-case.git
 
 SPEC CHECKSUMS:
-  FBSnapshotTestCase: 094f9f314decbabe373b87cc339bea235a63e07a
+  FBSnapshotTestCase: '094f9f314decbabe373b87cc339bea235a63e07a'
```

Comment:
No, it only affects those that start with zero. And it's about `psych` gem version IIRC, not ruby itself.

Also, great job!